### ISSUE TITLE
Spacing documentation missing CSS vars

### ISSUE THREAD

**Issue by dbecherdialpad** [dialpad/dialtone#332]
We should document the CSS variables (I think available as `var(--su{N})`) for spacing as well as the LESS vars: 
https://deploy-preview-150--dialtone.netlify.app/design/spacing/spacing-system/

**Comment by dialpad-drew** [comment]
Good call. Is there a reason to show the Less vars? Would we want to only document the CSS vars?

**Comment by dbecherdialpad** [comment]
> Is there a reason to show the Less vars? Would we want to only document the CSS vars?

This is a good question not just for spacing, but in general. I'm no expert but I think there's some performance overhead to using a custom property, so any time we can use a static value for that, it's preferable. This is why I usually go for the LESS var (though some of them output a css var anyway). None of the spacing, etc variables (and even a lot of the others) need to be "variable" so it's only really for convenience when writing plain CSS. For this ticket I was in a codepen where I didn't have LESS so maybe it's a moot point.
